builds/acr-cleanup/acr-registry-cleanup.yaml

```diff
@@ -39,6 +40,10 @@ parameters:
         olderThan: "14d"
         tagFilter: "^pr-.*"
         keep: 1
+      - repoRegex: '^labs/.*'
```

Comment:
@endakelly are you aware of other repos that we can delete?

Reply:
You could maybe clear down plum and toffee on a similar schedule since they are just test applications